src/mongo/db/query/explain.cpp

```diff
@@ -644,6 +653,16 @@ void Explain::generatePlannerInfo(PlanExecutor* exec,
     BSONObjBuilder plannerBob(out->subobjStart("queryPlanner"));
 
     plannerBob.append("plannerVersion", QueryPlanner::kPlannerVersion);
+
+    const auto mps = getMultiPlanStage(exec->getRootStage());
+    int originaWinningPlanIdx = static_cast<size_t>(mps->originalWinningPlanIdx());
+
+    if (originaWinningPlanIdx > -1) {
```

Comment:
It looks like you never use the actual index, so how about changing line 658 to be
`const bool usedBackupPlan = (static_cast<size_t>(mps->originalWinningPlanIdx()) > -1);`? That way you can collapse lines 660-664 into one line.